### DISCUSSION TITLE
Add discord webhooks to the list of things obscured in logs.

### DISCUSSION THREAD

**Discussion by birdman3131** [gameserverapp/Platform#870] [Suggestions]
Admin passwords and some other api keys are replaced with ***** in the logs GSA shows. 

Can we get discord webhooks added as well? 

Format of
https://discord.com/api/webhooks/1224824354290288467/ecxmPSf7kansafAykhvJpu1HSK5DRWD8bwWvTm3rqyVKnc1YXZB6iwFzLxSbO-97Kc2i
(Above webhook has been deleted.)

**Comment by mvaessen** [comment]
converted into dev item
